src/Components/Authentication/ReqResAuth/RestrictAuth.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useLocation, Outlet, Navigate } from 'react-router-dom'
+import { useAuth } from "../../../context/authContext";
+
+
+const RistrictAuth = () => {
+	const { authState: { userId } } = useAuth();
+	console.log(userId)
```

Comment:
I think, you can remove this console statement.

Reply:
Will do it!